src/test/App.test.js

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from '../App';
+import { BrowserRouter } from "react-router-dom";
+import { SnackbarProvider } from 'notistack';
 
```

Comment:
Maybe we can look into snapshot testing (I think this is simple-ish to setup using `jest` iirc) as a way to kickstart initial testing practices, and then develop into better tests, eventually using something like [`react-testing-library`](https://github.com/testing-library/react-testing-library) - I've used it a little, and it's pretty cool.

Reply:
Agreed

Reply:
As an encouragement measure, we could add code coverage rules

Reply:
> As an encouragement measure, we could add code coverage rules

Not ideal, tbh (as code coverage is not the full picture), but would make for some good incentive early on, I agree. Will track this

Reply:
Tracked: https://github.com/NIAEFEUP/nijobs-fe/issues/18